eva-accession-core/src/test/java/uk/ac/ebi/eva/accession/core/io/AccessionedVcfFactoryTest.java

```diff
@@ -0,0 +1,254 @@
+        VariantVcfFactory accessionedVariantFactory = new CoordinatesVcfFactory();
+        accessionedVariantFactory.setIncludeIds(true);
+
+        // test that an ID is properly read
+        checkIds(accessionedVariantFactory, "1\t1000\trs123\tC\tT\t.\t.\t.", Collections.singleton("rs123"));
+
+        // test that a missing ID ('.') is not added to the IDs set
+        checkIds(factory, "1\t1000\t.\tC\tT\t.\t.\t.", Collections.emptySet());
+        checkIds(accessionedVariantFactory, "1\t1000\trs123;.\tC\tT\t.\t.\t.", Collections.singleton("rs123"));
+
+        // test that the ';' is used as the ID separator (as of VCF 4.2)
+        checkIds(accessionedVariantFactory, "1\t1000\trs123;rs456\tC\tT\t.\t.\t.",
+                Stream.of("rs123", "rs456").collect(Collectors.toSet()));
+
+        // test that the ',' is NOT used as the ID separator (as of VCF 4.2)
+        checkIds(accessionedVariantFactory, "1\t1000\trs123,rs456\tC\tT\t.\t.\t.",
+                Collections.singleton("rs123,rs456"));
+    }
+
+    @Test
+    public void testVariantIdsDisabled() {
+        // ignore ids if explicitly configured, to comply with the interface
+        VariantVcfFactory nonAccessionedVariantFactory = new CoordinatesVcfFactory();
+        nonAccessionedVariantFactory.setIncludeIds(false);
+        Set<String> emptySet = Collections.emptySet();
+
+        // test that an ID is ignored
+        checkIds(nonAccessionedVariantFactory, "1\t1000\trs123\tC\tT\t.\t.\t.", emptySet);
+
+        // test that several ID are ignored
+        checkIds(nonAccessionedVariantFactory, "1\t1000\trs123;rs456\tC\tT\t.\t.\t.", emptySet);
+
+        // test that a missing ID ('.') is not added to the IDs set
+        checkIds(nonAccessionedVariantFactory, "1\t1000\t.\tC\tT\t.\t.\t.", emptySet);
+    }
+
+    private void checkIds(VariantVcfFactory variantVcfFactory, String vcfLine, Set<String> expectedIds) {
+        List<Variant> expectedVariants = new LinkedList<>();
+        expectedVariants.add(new Variant("1", 1000, 1000, "C", "T"));
+        expectedVariants.get(0).setIds(expectedIds);
+
+        List<Variant> parsedVariants = variantVcfFactory.create(FILE_ID, STUDY_ID, vcfLine);
+
+        assertEquals(expectedVariants, parsedVariants);
+        assertEquals(expectedIds, parsedVariants.get(0).getIds());
+    }
+
+    @Test
+    public void testReadContextBaseVariant() {
+        String line = "chr1\t1000\t.\tGCAG\tG\t.\t.\t.";
+        List<Variant> expectedResult = Collections.singletonList(new Variant("chr1", 1001, 1003, "CAG", ""));
+        List<Variant> actualResult = factory.create(FILE_ID, STUDY_ID, line);
+        assertEquals(expectedResult, actualResult);
+
+        line = "chr1\t1000\t.\tG\tGG\t.\t.\t.";
+        expectedResult = Collections.singletonList(new Variant("chr1", 1001, 1001, "", "G"));
+        actualResult = factory.create(FILE_ID, STUDY_ID, line);
+        assertEquals(expectedResult, actualResult);
```

Comment:
Does [this test](https://github.com/EBIvariation/eva-accession/blob/73a433cc584902edf692f032258e5f4f6f528753/eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/batch/tasklets/reportCheck/ReportCheckTaskletTest.java#L196) still pass? If so, could you investigate why?

Reply:
This test was still using CoordinatesVcfFactory instead of the newly created Accessioned VCFFactory. 
Checked and Updated the test cases in **ReportCheckTaskletTest**